charts/spire/charts/spire-server/templates/tests/test-connection.yaml

```diff
@@ -11,17 +11,17 @@ spec:
   securityContext:
     {{- toYaml .Values.podSecurityContext | nindent 4 }}
   containers:
-    - name: wget
-      image: busybox
+    - name: netcat
+      image: cgr.dev/chainguard/busybox:latest-glibc
       command: ['nc']
       args: ['-zvw3', '{{ include "spire-server.fullname" . }}', '{{ .Values.service.port }}']
       securityContext:
         {{- toYaml .Values.securityContext | nindent 8 }}
     {{- if eq (.Values.federation.enabled | toString) "true" }}
-    - name: wget-federation-bundle-endpoint
-      image: busybox
-      command: ['wget']
-      args: ['--no-check-certificate', '-O', '/dev/null', 'https://{{ include "spire-server.fullname" . }}.{{ include "spire-server.namespace" . }}.svc.cluster.local:{{ .Values.federation.bundleEndpoint.port }}']
+    - name: netcat-federation-bundle-endpoint
+      image: cgr.dev/chainguard/busybox:latest-glibc
+      command: ['nc']
+      args: ['-zvw3', '{{ include "spire-server.fullname" . }}.{{ include "spire-server.namespace" . }}', '{{ .Values.federation.bundleEndpoint.port }}']
```

Comment:
Its failing because chainguard busybox's wget is crippled to not be able to speak https properly.... curl would fix this and actually test the service. -1 to crippling the test by switching it to nc.

Reply:
I dont think any functionality is lost because we werent looking at the resultant file before, just sending it to /dev/null. If we want a test that looks at http status or the actual file we can add that seperately.